### PR TITLE
Accept primary color to be configured

### PR DESCRIPTION
**Pull request by jamiebuilds**
**Type:** Feature

- Issue: https://github.com/verdaccio/ui/issues/11
- Corresponding PR: https://github.com/verdaccio/verdaccio/pull/1282

### CODE REVIEW

File: src/webui/utils/styles/colors.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ const colors = {
     // Main colors
     // -------------------------
 
-    primary: '#4b5e40',
+    primary: window.PRIMARY_COLOR || '#4b5e40',
```
Comment (juanpicado) [inline src/webui/utils/styles/colors.js]: small mistake here, it should be `VERDACCIO_PRIMARY_COLOR`

**Comment by jamiebuilds** [comment]
Updated :)

**Comment by jamiebuilds** [comment]
Added to docs: https://github.com/verdaccio/website/pull/79

File: README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ yarn dev
 ```
 The configuration file is located on `tools/_config.yaml`.
 
-Run lintings and test to check your code is clean.
+Run linting tooling and test to check your code is clean before commit.
```
Comment (juanpicado) [inline README.md]: Thanks :-)

**Comment by juanpicado** [comment]
Thanks @jamiebuilds !

**Pull request merged**